modules/sdk/ant-bnd/src/main/java/com/liferay/ant/bnd/jsp/JspAnalyzerPlugin.java

```diff
@@ -328,7 +328,8 @@ protected void addTaglibRequirements(
 			// Check to see if the JAR provides this TLD itself which would
 			// indicate that it already has access to the required classes
 
-			if (containsTLD(analyzer, analyzer.getJar(), "META-INF", uri) ||
+			if (containsTLD(
+					analyzer, analyzer.getJar(), "META-INF/resources", uri) ||
 				containsTLD(analyzer, analyzer.getJar(), "WEB-INF/tld", uri) ||
 				containsTLDInBundleClassPath(analyzer, "META-INF", uri)) {
```

Comment:
@csierra - Should the change be applied here too?

Reply:
@petershin hey... I am not sure... I just applied the minimum change to exemplify the problem I found...

Reply:
@csierra - ray added this code, so I'll check with him